ENTITY/SetEntityVisible.md

```diff
@@ -9,6 +9,8 @@ void SET_ENTITY_VISIBLE(Entity entity, BOOL toggle, BOOL unk);
 ```
 
 Toggle the visibility of a given entity.
+If you want to change visibility of `PlayerPedId()`, it will change for other players and **will not** apply for you.
+You have to use `SET_ENTITY_LOCALLY_INVISIBLE` or other natives like this to change visibility of your character.
```

Comment:
Looks good, you should make `SET_ENTITY_LOCALLY_VISIBLE` a reference though (which creates a link), for example:
`[`SetEntityLocallyInvisible`](#0xE135A9FF3F5D05D8)` that way users can find how to use the native you just told them to use with minimal effort by clicking on it :-)